experimental/rsocket-src/RSocketRequester.cpp

```diff
@@ -69,28 +66,26 @@ RSocketRequester::requestStream(Payload request) {
     request = std::move(request),
     srs = reactiveSocket_
   ](yarpl::Reference<yarpl::flowable::Subscriber<Payload>> subscriber) mutable {
-    // TODO eliminate OldToNew bridge
-    auto os = std::make_shared<OldToNewSubscriber>(std::move(subscriber));
     eb->runInEventBaseThread([
       request = std::move(request),
-      os = std::move(os),
+      subscriber = std::move(subscriber),
       srs = std::move(srs)
-    ]() mutable { srs->requestStream(std::move(request), std::move(os)); });
+    ]() mutable { srs->requestStream(std::move(request), std::move(subscriber)); });
   });
 }
 
 yarpl::Reference<yarpl::single::Single<reactivesocket::Payload>>
 RSocketRequester::requestResponse(Payload request) {
   // TODO bridge in use until SingleSubscriber is used internally
-  class SingleToSubscriberBridge : public Subscriber<Payload> {
+  class SingleToSubscriberBridge : public yarpl::flowable::Subscriber<Payload> {
```

Comment:
After this change lands we can change requestResponse to use SingleObserver instead of FlowableSubscriber.